core/arch/arm/plat-hikey/main.c

```diff
@@ -205,4 +206,47 @@ static TEE_Result peripherals_init(void)
 	return TEE_SUCCESS;
 }
 
+/*
+ * First time, add low 16 bits of tsc as entropy.
+ * Subsequently, accumulate 2 low bits each time
+ * and add entropy when we reach 8 bits.
+ */
+
+void plat_prng_add_jitter_entropy(void)
```

Comment:
That's more or less what `inject_entropy_with_timestamp()` does (`entry_std.c`), no? Either one or the other but we don't want two ways of adding time-based entropy.